src/examples/com/ibm/cloud/ibm_key_protect_api/v2/KeyAliasExample.java

```diff
@@ -0,0 +1,103 @@
+ */
+
+package com.ibm.cloud.ibm_key_protect_api.v2;
+
+import com.ibm.cloud.ibm_key_protect_api.v2.model.KeyWithPayload;
+import com.ibm.cloud.sdk.core.security.IamAuthenticator;
+import com.ibm.cloud.sdk.core.service.exception.ServiceResponseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+//
+// This class provides examples of how to use the Key Protect key alias services.
+//
+// The following configuration properties are assumed to be exported as environment variables
+//
+// IBMCLOUD_API_KEY=<IBM Cloud APIKEY for the User>
+// KP_INSTANCE_ID=<ID of the service instance to be used >
+// IAM_AUTH_URL=<IAM Token Service URL>
+// KP_SERVICE_URL=<Service URL>
+//
+// API docs: https://cloud.ibm.com/apidocs/key-protect
+
+public class KeyAliasExample {
+
+    private static final Logger logger = LoggerFactory.getLogger(KeyAliasExample.class);
+
+    //values to be read from the env setting
+    private static String ibmCloudApiKey;
+    private static String exampleInstance;
+    private static String iamAuthUrl;
+    private static String serviceUrl;
+
+    static {
+        Map<String, String> config = System.getenv();
+        ibmCloudApiKey = config.get("IBMCLOUD_API_KEY");
+        exampleInstance = config.get("KP_INSTANCE_ID");
+        iamAuthUrl = config.get("IAM_AUTH_URL");
+        serviceUrl = config.get("KP_SERVICE_URL");
+    }
+
+    public KeyAliasExample() {
+    }
+
+    public static void main(String[] args) {
+        IamAuthenticator authenticator = new IamAuthenticator(ibmCloudApiKey);
+        authenticator.setURL(iamAuthUrl);
+        authenticator.validate();
+        // payload is base64 encoded string of "It is a really important message"
+        String payload = "SXQgaXMgYSByZWFsbHkgaW1wb3J0YW50IG1lc3NhZ2U=";
+        String keyAlias = "sdkKeyAlias";
+        String keyName = "sdk-created-key";
+        String keyDesc = "created via sdk";
+
+        try {
+            IbmKeyProtectApi exampleService = IbmKeyProtectApi.newInstance(authenticator);
+            exampleService.setServiceUrl(serviceUrl);
+
+            // Create key
+            logger.info("Create a key");
+            String keyId = KpUtils.createKey(exampleService, exampleInstance, keyName, keyDesc,
+                    payload, false);
+            logger.info(String.format("Key with ID %s created", keyId));
+
+            // Create key alias
+            logger.info("Create a key alias");
+            KpUtils.createKeyAlias(exampleService, exampleInstance, keyId, keyAlias);
+            logger.info(String.format("Key alias %s for key with ID %s created", keyAlias, keyId));
+
+            // Get a key using key alias
+            logger.info("Get a key using key alias");
+            KeyWithPayload keyWithPayload = KpUtils.getKey(exampleService, exampleInstance, keyAlias);
+            logger.info(String.format("Got key with alias %s: ", keyAlias) + keyWithPayload);
```

Comment:
Lets not print the payload, since that could end up being copy-pasted accidentally